### ISSUE TITLE
Sensitive information storage and it's access/visibility

### ISSUE THREAD

**Issue by chinmaybhatk** [frappe/erpnext#50700]
Hi,
For better control on the fields and it's accessibility, we should have a property to each component to mark whether it's sensitive data or non sensitive data along with its sensitivity level.

Ref: https://knowledge.hubspot.com/properties/store-sensitive-data

**Comment by DipenFrappe** [comment]
https://docs.frappe.io/framework/data-masking
https://github.com/frappe/frappe/pull/32684
